### PR TITLE
Criminal

### LINKED ISSUE #5: Create Criminal
Create entity, repository, service and controller (CURD operations)

### PR DESCRIPTION
**Pull request by helenahalldiniths**
This will fix #5

### CODE REVIEW

File: src/main/java/se/iths/crimedatabase/entity/Criminal.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import javax.persistence.Id;
+
+@Entity
+public class Criminal {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String firstName;
+    private String lastName;
+
```
Comment (LordRekishi) [inline src/main/java/se/iths/crimedatabase/entity/Criminal.java:17]: I suggest we add: LocalDate dateOfBirth as a field here

**Review by LordRekishi — CHANGES_REQUESTED** [review]

File: src/main/java/se/iths/crimedatabase/entity/Criminal.java (hunk 1 above)
Comment (helenahalldiniths) [reply src/main/java/se/iths/crimedatabase/entity/Criminal.java:17]: Fine by me!

**Review by FelixJacobsen — APPROVED** [review]
Looks good and follows our Entity structure

**Review by LordRekishi — APPROVED** [review]
Looks great!

**Pull request merged**